dependencies-path.yaml

```diff
@@ -0,0 +1,54 @@
+- dependency: "etcd"
+  refPaths:
+  - build/root/WORKSPACE
+  - cluster/gce/config-test.sh
+  - cluster/gce/manifests/etcd-empty-dir-cleanup.yaml
+  - cluster/gce/manifests/etcd.manifest
+  - cluster/gce/upgrade-aliases.sh
+  - cluster/images/etcd-empty-dir-cleanup/Makefile
```

Comment:
i'd be really happy if the etcd image is moved to build/images at this point.

Reply:
+1 to open a new bug for that